packages/ui/src/modal/bid/BidModal.tsx

```diff
@@ -765,13 +765,22 @@ export function BidModal({
                           }}
                         >
                           <Button
+                            disabled={providerOptionsContext.disableJumperLink}
                             css={{ flex: '1 0 auto' }}
                             color="secondary"
                             onClick={() => {
                               window.open(convertLink, '_blank')
                             }}
                           >
-                            {copy.ctaConvertManually}
+                            {providerOptionsContext.disableJumperLink
+                              ? copy.ctaBid.length > 0
+                                ? copy.ctaBid
```

Comment:
Maybe we should make this a variable so we can reuse in both the buttons? Referring to this piece:
```
copy.ctaBid.length > 0
                          ? copy.ctaBid
                          : token && token.token
                          ? 'Make an Offer'
                          : trait
                          ? 'Make an Attribute Offer'
                          : 'Make a Collection Offer'